src/main/scala/com/lightbend/lagom/docs/DocumentationGenerator.scala

```diff
@@ -374,6 +374,16 @@ case class Version(name: String, languages: Seq[LanguageVersion]) {
   */
 case class LanguageVersion(name: String, language: String, file: File, toc: TOC) {
   def pageFor(path: String) = VersionPage(name, toc.mappings.get(path).isDefined)
+
+  private val apiRoot = (name, language) match {
+    case (_, "java") => "api/index.html"
+    // Note that this version check will break and need to be updated if we ever have a 1.10+
```

Comment:
Please forgive this... I didn't want to complicate things even further! We have time to fix this if it ever happens… and more likely that we'll move to 2.0 before then anyway. :smile:

Reply:
I don't get your point her, but you are forgiven.

Reply:
This is really embarrassing and your future self will regret this decision. :trollface:

Reply:
I'm hoping to be long gone by then so it can be someone else's problem 😉